Sources/MultipartKit/FormDataEncoder/FormDataEncoder.swift

```diff
@@ -26,6 +27,20 @@ public struct FormDataEncoder: Sendable {
     public func encode<E: Encodable>(_ encodable: E, boundary: String) throws -> String {
         try MultipartSerializer().serialize(parts: parts(from: encodable), boundary: boundary)
     }
+    
+    /// Encodes an `Encodable` item to `Data` using the supplied boundary.
+    ///
+    ///     let a = Foo(string: "a", int: 42, double: 3.14, array: [1, 2, 3])
+    ///     let data = try FormDataEncoder().encodeToData(a, boundary: "123")
+    ///
+    /// - parameters:
+    ///     - encodable: Generic `Encodable` item.
+    ///     - boundary: Multipart boundary to use for encoding. This must not appear anywhere in the encoded data.
+    /// - throws: Any errors encoding the model with `Codable` or serializing the data.
+    /// - returns: `multipart/form-data`-encoded `String`.
```

Comment:
```suggestion
    /// - Parameters:
    ///     - encodable: An `Encodable` item.
    ///     - boundary: The multipart boundary to use for encoding. This string must not appear in the encoded data.
    /// - Throws: Any errors encoding the model or serializing the data.
    /// - Returns: A `multipart/form-data`-encoded `String`.
```

This better matches the style of documentation comments in use elsewhere (or at least anywhere that I've been writing them 😅).

(And of course, update the comments for the other methods you've added accordingly.)